web/app/themes/clarity/inc/admin/prior-party/prior-party-banner-admin.php

```diff
@@ -93,6 +104,11 @@ public function __construct()
         // Add the delete action to the schedule.
         add_action('prior_party_banner_event_cleanup_cron_hook', [$this, 'deleteOldTrackEvents']);
 
+        // Ensure administrator always has the required capability.
+        if (current_user_can('administrator')) {
+            $this->required_capability = 'administrator';
+        }
```

Comment:
@EarthlingDavey Is this necessary? Should `private string $required_capability = 'manage_prior_party_banners';` not just be set to 'administrator' instead?

Reply:
Or are there cases where non-admin users are give the prior_party_banners_capability?

Reply:
Oh, is there a difference between administrator and agency_admin? In which case I've answered my own question 😅